src/makeRobotAccountant.js

```diff
@@ -6,7 +6,16 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let timesCalled = 0;
+
+  return firstAddendum =>
+    secondAddendum => {
+      timesCalled++;
+
+      return timesCalled <= 3 || timesCalled % 2
+        ? firstAddendum + secondAddendum
+        : 'Bzzz... Error!';
+    };
 }
```

Comment:
```suggestion
  return firstAddendum => (
    secondAddendum => {
      timesCalled++;

      return timesCalled <= 3 || timesCalled % 2
        ? firstAddendum + secondAddendum
        : 'Bzzz... Error!';
    };
  )
}
```

Reply:
Eslint forbids this, saying "Unnecessary parentheses around expression"